s/soap-builder/soap-builder_ubi8.4.sh

```diff
@@ -0,0 +1,49 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : soap-builder
+# Version       : 1.0.0-SNAPSHOT
+# Source repo   : https://github.com/reficio/soap-ws
+# Tested on     : ubi: 8.4
+# Script License: Apache License 2.0
+# Maintainer's  : Sapna Shukla<Sapna.Shukla@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+#Variables.
+BASE_PACKAGE_NAME=soap-ws
+PACKAGE_NAME=soap-builder
+PACKAGE_URL=https://github.com/reficio/soap-ws
+PACKAGE_VERSION=master
```

Comment:
The versions in header and here (`master`) do not match. In case of top-of-tree i.e. master, please mention a specific commit # against which the script has been verified.

Reply:
Yes, changed the version from master to the commit ID from the top of the tree.